src/BuildScriptGenerator/DefaultDockerfileGenerator.cs

```diff
@@ -255,5 +261,50 @@ private string ConvertToRuntimeVersion(string platformName, string platformVersi
         {
             return this.platformDetector.GetCompatiblePlatforms(ctx);
         }
+
+        private void ValidateDockerfileImages(DockerfileProperties dockerfileProperties)
+        {
+            if (string.IsNullOrEmpty(dockerfileProperties.BuildImageName))
+            {
+                var message = "Provided build image name parsed from the provided --build-image argument is empty. " +
+                    "Please provide a valid value for --build-image, or remove the argument and use the default " +
```

Comment:
Should this read "...remove the argument **to** use the default..." instead of "...remove the argument **and** use the default..."